src/network/src/esp/src/NetworkConfig.cpp

```diff
@@ -19,10 +32,16 @@ wifi_config_t* NetworkConfig::getDefaultNetworkConfig() {
     s_wifiConfig.sta.sort_method = WIFI_CONNECT_AP_BY_SIGNAL;
     s_wifiConfig.sta.threshold.rssi = INT8_MIN;
     s_wifiConfig.sta.threshold.authmode = DEFAULT_AUTH_MODE;
+
     return &s_wifiConfig;
 }
 
-esp_interface_t NetworkConfig::getInterface() { return ESP_IF_WIFI_STA; }
+esp_interface_t NetworkConfig::getInterface() {
+    if (gs_isRouter) {
```

Comment:
Just a personnal preference, I prefer to user conditionnal operator for readability in this case.
"return gs_isRouter ? ESP_IF_WIFI_AP : ESP_IF_WIFI_STA;"